packages/core/src/components/html-attributes.ts

```diff
@@ -19,3 +19,13 @@ export function normalizeBooleanAttribute(attributeName: string, value: boolean
 
   return ret;
 }
+
+export function normalizeNumberAttribute(val: number | string): number | undefined | null {
+  if (val === undefined || val === '' || val === 'undefined' || val === 'NaN') {
```

Comment:
Can you give a bit background on the comparison against `'undefined'` and `'NaN'`

Reply:
There is no background. Can be removed

Reply:
@bigopon what's your suggestion?

Reply:
Lets start with this. I think if it comes from your realworld usage then its fine to start with. If it proves to be against common expectation, we can change it